### PR TITLE
Add JDBC URL parameter for MySQL

### PR DESCRIPTION
**Pull request by Jessie212**

### CODE REVIEW

File: docs/src/main/sphinx/connector/mysql.rst [hunk 1]
```diff
@@ -27,10 +27,23 @@ connection properties as appropriate for your setup:
 .. code-block:: text
 
     connector.name=mysql
-    connection-url=jdbc:mysql://example.net:3306
+    connection-url=jdbc:mysql://example.net:3306?serverTimezone=UTC
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/mysql.rst:30]: remove this change here

File: docs/src/main/sphinx/connector/mysql.rst [hunk 2]
```diff
@@ -27,10 +27,23 @@ connection properties as appropriate for your setup:
...
     connection-user=root
     connection-password=secret
 
+The ``connection-url`` defines the connection information and parameters to pass
+to the MySQL JDBC driver. The supported parameters for the URL are
+available in the `MySQL Developer Guide
+<https://dev.mysql.com/doc/connector-j/8.0/en/>`_.
+
+For known issues and limitations, see `known issues and limitations
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/mysql.rst:39]: change this to be talking about connetion url parameters. Something like

```
For example, you can configure the JDBC driver to interpret any time values based on UTC as 
timezone on the server as `workaround for a known issue <>`_ with the following ``connection-url``:

connection-url=jdbc:mysql://example.net:3306?serverTimezone=UTC
```

**Review by mosabua — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/connector/mysql.rst [hunk 3]
```diff
@@ -31,6 +31,23 @@ connection properties as appropriate for your setup:
     connection-user=root
     connection-password=secret
 
+The ``connection-url`` defines the connection information and parameters to pass
+to the MySQL JDBC driver. The supported parameters for the URL are
+available in the `MySQL Developer Guide
+<https://dev.mysql.com/doc/connector-j/8.0/en/>`_.
+
+For example, using the following ``connection-url``:
+connection-url=jdbc:mysql://example.net:3306?serverTimezone=UTC allows you to
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/mysql.rst:40]: move this into a code block ..

File: docs/src/main/sphinx/connector/mysql.rst [hunk 4]
```diff
@@ -31,6 +31,23 @@ connection properties as appropriate for your setup:
...
+configure the JDBC driver to interpret time values based on UTC as a timezone on
+the server and serves as a `workaround for a known issue
+<https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-usagenotes-known-issues-limitations.html>`_.
+
+
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/mysql.rst:45]: code block here

**Review by mosabua — APPROVED** [review]
Looks good now. Text is mostly identical to the existing (already approved) text in sqlserver.rst

**Review by ebyhr — APPROVED** [review]

**Comment by ebyhr** [comment]
Merged, thanks!

**Pull request merged**